docs/index.md

```diff
@@ -1,64 +1,27 @@
-# Iterative Provider
+# Terraform Provider Iterative (TPI)
 
-Use the Iterative Provider to launch resource-intensive tasks in popular cloud providers with a single Terraform file.
+![TPI](https://static.iterative.ai/img/cml/banner-terraform.png)
```

Comment:
Broken, it seems

![image](https://user-images.githubusercontent.com/1477535/159874388-da5f10de-d185-4f38-be5a-c1ab6f7fb357.png)

Reply:
Yes, should be https://static.iterative.ai/img/cml/banner-tpi.svg

Reply:
Instead of https://static.iterative.ai/img/cml/banner-terraform.png